### PR TITLE
GLT-1035 confirm that LibraryDesigns correspond to correct sample classes

### PR DESCRIPTION
**Pull request by callunity**

### CODE REVIEW

File: migration/src/main/java/uk/ac/bbsrc/tgac/miso/migration/destination/ValueTypeLookup.java [hunk 1]
```diff
@@ -68,7 +68,7 @@
   private Map<Long, LibraryType> libraryTypeById;
   private Map<String, Map<String, LibraryType>> libraryTypeByPlatformAndDescription;
   private Map<Long, LibraryDesign> libraryDesignById;
-  private Map<String, LibraryDesign> libraryDesignByName;
+  private Map<String, Map<String, LibraryDesign>> libraryDesignBySampleClassAliasAndName;
```
Comment (apmasell) [inline migration/src/main/java/uk/ac/bbsrc/tgac/miso/migration/destination/ValueTypeLookup.java]: Doesn't this belong in the main repo?

Comment (callunity) [reply migration/src/main/java/uk/ac/bbsrc/tgac/miso/migration/destination/ValueTypeLookup.java]: It sure does. Fixed!

**Review by apmasell — APPROVED** [review]

**Review by apmasell — APPROVED** [review]

**Comment by tonydebat** [comment]
I get the following error when migrating with the latest stage db.

```
[ERROR] Failed to execute goal org.flywaydb:flyway-maven-plugin:3.2.1:migrate (default-cli) on project sqlstore: org.flywaydb.core.internal.dbsupport.FlywaySqlScriptException:
[ERROR] Migration V8251__add_more_libraryDesigns.sql failed
[ERROR] ---------------------------------------------------
[ERROR] SQL State  : 42S02
[ERROR] Error Code : 1146
[ERROR] Message    : Table 'miso.librarydesigncode' doesn't exist
[ERROR] Location   : db/migration/V8251__add_more_libraryDesigns.sql (/Users/tdebat/code/gsi/miso-lims/sqlstore/target/classes/db/migration/V8251__add_more_libraryDesigns.sql)
[ERROR] Line       : 2
[ERROR] Statement  : INSERT INTO LibraryDesignCode (code, description) VALUES ("TR", "TR")
[ERROR] -> [Help 1]
[ERROR]
[ERROR] To see the full stack trace of the errors, re-run Maven with the -e switch.
[ERROR] Re-run Maven using the -X switch to enable full debug logging.
[ERROR]
[ERROR] For more information about the errors and possible solutions, please read the following articles:
[ERROR] [Help 1] http://cwiki.apache.org/confluence/display/MAVEN/MojoExecutionException
```

**Comment by callunity** [comment]
Sorry, I should have mentioned that this depends on https://github.com/TGAC/miso-lims/pull/522 to run first (it just got merged into develop).

**Review by tonydebat — APPROVED** [review]

**Pull request merged**